solidity/test/FixedRatePoolConverter.js

```diff
@@ -164,7 +164,7 @@ describe('FixedRatePoolConverter', () => {
         await reserveToken.approve(converter.address, value, { from: sender });
         await reserveToken2.approve(converter.address, value, { from: sender });
 
-        const res = await converter.addLiquidity(
+        const res = await converter.methods['addLiquidity(address[2],uint256[2],uint256)'](
```

Comment:
The usual approach is to implement a differently named "override" in the test helper contract, instead of explicitly working with selectors. Could you refactor this away?

Reply:
There's another trick BTW.

In the actual test, the Truffle Contract object (`converter`) has the following fields:
1. `converter.methods['addLiquidity(address[2],uint256[2],uint256)']`
2. `converter.methods['addLiquidity(uint256,uint256,uint256)']`
3. `converter.addLiquidity`

When the Truffle Contract object is created, the 3rd field is set to "point" to either one of the other fields (depending on how the ABI was built).

So immediately after creating it, we can override this, and set it to "point" to the one that we intend to call (assuming that we intend to call only one of them, as is the case at present).

Subsequently, we can use  `converter.addLiquidity` in the usual ("natural") manner.

I didn't use this option, because we create the Truffle Contract object in several different places, so the benefit of keeping our code clean is diminished.

Reply:
I totally understand the problem, it's just that the solution is too "low level"

Reply:
@lbeder : This request is revoked, correct?

Reply:
Correct